README.md

```diff
@@ -97,29 +100,33 @@ In addition to using Broccoli via the combination of `broccoli-cli` and a `Brocf
 By way of example, let's assume we have a graph of Broccoli nodes constructed via a combination of `Funnel` and `MergeTrees`:
 
 ```js
-const html = new Funnel(appRoot, {
+// non Brocfile.js, regular commonjs
+const funnel = require('broccoli-funnel');
+const merge = require('broccoli-merge');
+
+const html = funnel(appRoot, {
   files: ['index.html'],
   annotation: 'Index file'
 })
 
-const js = new Funnel(appRoot, {
+const js = funnel(appRoot, {
```

Comment:
These changes (away from `new Foo`) seem unrelated to ESM, lets do them in a different PR.

Reply:
sure thing